packages/pwa-kit-create-app/program.json

```diff
@@ -182,28 +214,23 @@
                 "type": "object",
                 "properties": {
                     "id": {
-                        "type": "string",
-                        "description": "The unique identifier for the validator."
```

Comment:
We mentioned this during the call. I don't see the `description` info captured anywhere else. Can we keep the "description" info since it's seems valuable info?